cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -1,3 +1,4 @@
+/* global _ */
```

Comment:
@bjacobel Should `_` be added to globals dict in [eslint config file](https://github.com/edx/edx-platform/blob/master/.eslintrc.json#L3-L22) ? What is the best practice here ?